packages/wallets/src/features/cashier/modules/WithdrawalVerification/__tests__/WithdrawalVerification.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useCountdown } from 'usehooks-ts';
+import { useSettings, useVerifyEmail } from '@deriv/api';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import WithdrawalVerification from '../WithdrawalVerification';
+import '@testing-library/jest-dom';
+
+jest.mock('../WithdrawalVerificationRequest', () => ({
+    ...jest.requireActual('../WithdrawalVerificationRequest'),
+}));
+
+jest.mock('../WithdrawalVerificationSent', () => ({
```

Comment:
same as above,

Reply:
replied above https://github.com/binary-com/deriv-app/pull/12417#discussion_r1433918760 🙏

Reply:
Updated and removed unneeded mock